content/guides/hmr-react.md

```diff
@@ -157,6 +157,21 @@ module.exports = {
 };
 ```
 
+Important to note:
+
+If you have a `query` section for the `babel-loader` in your `rules`, remember to turn off the `modules` tranformation as well, otherwise HMR won't work.
+
+```js
+loader: "babel-loader",
+query: {
```

Comment:
Maybe you could rewrite the idea as a warning (`W>`). Perfect for secondary info.